tests/functional-tests/README.md

```diff
@@ -0,0 +1,77 @@
+# IBM Spectrum Scale CSI - Automation
+
+The IBM Spectrum Scale Container Storage Interface (CSI) project enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
+
+This project contains an automation framework to exercise the IBM Spectrum Scale CSI Driver and Operator functionality .
+
+
+### Testbed environment 
+
+- IBM Spectrum Scale Cluster - 5.0.4.1+ Version  (**IBM Spectrum Scale supported kernel version**)
+- Kubernetes Cluster Version 1.14 - 1.18
+- Openshift Version 4.2.x,4.3.x
+- IBM Spectrum Scale Cluster CSI Version - 1.1.0+
+
+### Pre-requesite for automation framework
+
+Install Python (3.7.4 or higher) and below mentioned pip modules:
+
+  ``` 
+        python3.7 -m pip install kubernetes
+        python3.7 -m pip install pytest
+        python3.7 -m pip install pytest-html
+        python3.7 -m pip install jsmin
+  ```
+       
+
+### How to run IBM Spectrum Scale CSI test automation
+- Clone the source code from [git](https://github.com/IBM/ibm-spectrum-scale-csi/) repository.
+
+- Configure parameters in [tests/functional-tests/config.json](https://github.com/IBM/ibm-spectrum-scale-csi/blob/dev/tests/functional-tests/config.json) file. 
```

Comment:
Should be using relative links...  otherwise you will always force the URL to link to the dev branch, not the branch you are sitting on.  `[test...](./tests/functional-test/config.json)`